common/src/main/java/com/wynntils/features/debug/ItemDebugTooltipsFeature.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.debug;
+
+import com.wynntils.core.features.DebugFeature;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.utils.KeyboardUtils;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wynn.handleditems.ItemModel;
+import com.wynntils.wynn.handleditems.WynnItem;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+public class ItemDebugTooltipsFeature extends DebugFeature {
```

Comment:
What does this do exactly? How is this different from that other, item related debug feature?

Reply:
<img width="541" alt="Screen Shot 2022-12-31 at 16 15 44" src="https://user-images.githubusercontent.com/6209108/210141357-cbe7501b-411f-4f45-b75e-bc2338cb7da7.png">

Reply:
It is mostly to showcase the new framework, but I have also had some use of it when debugging. It will basically show you everything Wynntils knows about the item. If it is too long, it will be cut off unless you press right shift, since gear items with lots of identification can easily fill the screen otherwise.